cloudigrade/account/models.py

```diff
@@ -0,0 +1,15 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+from util.models import BaseModel
+
+
+class Account(BaseModel):
+    """Account model."""
+
+    account_id = models.DecimalField(
+        max_digits=12,
+        decimal_places=0,
+        db_index=True
+    )
+    account_arn = models.CharField(max_length=256)
```

Comment:
We probably want an index on this field too since we'll have lookups on it.

Reply:
and `unique=True`… though I'm not sure if adding that will automagically give it an index.

Reply:
Postgres will definitely give you a b-tree index on unique constraint.